packages/idyll-components/src/scroller.js

```diff
@@ -52,9 +52,11 @@ class Scroller extends React.Component {
       .setup({
         step: `#idyll-scroll-${this.id} .idyll-step`, // required
         container: `#idyll-scroll-${this.id}`, // required (for sticky)
-        graphic: `#idyll-scroll-${this.id} .idyll-scroll-graphic` // required (for sticky)
+        graphic: `#idyll-scroll-${this.id} .idyll-scroll-graphic`, // required (for sticky)
+        progress: true
```

Comment:
What is the performance implication of this if the user doesn't care about tracking scroll progress? Should we only set `progress` to true if they've bound a variable to the `progress` property?